training/DeepSpeed-Domino/megatron/checkpointing.py

```diff
@@ -503,6 +509,8 @@ def load_checkpoint(model, optimizer, opt_param_scheduler, load_arg='load', stri
     load_dir = getattr(args, load_arg)
 
     model = unwrap_model(model)
+    model_module = model.module
+    model = [model_module]
 
     state_dict, checkpoint_name, release = _load_base_checkpoint(load_dir, rank0=False)
 
```

Comment:
There is no `exit_on_missing_checkpoint` argument now. It will produce the following error if one has argument `load` but no checkpoint. 
```
[rank1]:     if not args.exit_on_missing_checkpoint:
[rank1]: AttributeError: 'Namespace' object has no attribute 'exit_on_missing_checkpoint'
```